src/unix/pipe.c

```diff
@@ -206,14 +206,20 @@ void uv_pipe_connect(uv_connect_t* req,
 }
 
 
-int uv_pipe_getsockname(const uv_pipe_t* handle, char* buffer, size_t* size) {
+typedef int (*peersockfunc)(int, struct sockaddr*, socklen_t*);
```

Comment:
Is the name acceptable?

we have sockname and peername, so peersockfunc was the best I could come up with.